geneticengine/core/grammar.py

```diff
@@ -298,7 +313,7 @@ def extract_grammar(
         - The grammar
 
     """
-    g = Grammar(starting_symbol, considered_subtypes)
+    g = Grammar(starting_symbol, expansion_depthing, considered_subtypes)
```

Comment:
This does not seem correct. expansion_depthning should be an extra parameter (to the end of the Grammar constructor parameters.)

Reply:
Does this fix it?